renderer/views/torrent-list.js

```diff
@@ -6,18 +6,18 @@ var hx = hyperx(h)
 var prettyBytes = require('pretty-bytes')
 
 function TorrentList (state, dispatch) {
-  var torrents = state.view.client
-    ? state.view.client.torrents
+  var torrents = state.client
+    ? state.client.torrents
```

Comment:
this seems to have come back from a merge, thought i deleted this. It's not necessary. `state.client.torrents` will always exist